Update scm-support.md (#2227)
semgrep/semgrep-docs: docs/getting-started/scm-support.md

| Bitbucket Cloud Free | <ul><li>Semgrep Assistant†</li><li> Semgrep Managed Scan†</li><li> Query console</li><li>Auto PRs for Supply Chain findings</li><li>Generic secrets (requires Semgrep Assistant)</li></ul> |
| Bitbucket Cloud Standard | <ul><li>Semgrep Assistant†</li><li>Semgrep Managed Scan†</li><li> Query console</li><li>Auto PRs for Supply Chain findings</li><li>Generic secrets (requires Semgrep Assistant)</li></ul> |
| Bitbucket Cloud Premium | <ul><li>Query console</li><li>Auto PRs for Supply Chain findings</li></ul> |

> | Bitbucket Data Center | <ul><li>Query console</li><li>Only versions 7.17+ are supported.</li><li>Diff-aware scans require Bitbucket Data Center version 8.8 or later.</li><li>Diff-aware scans to support versions prior to 8.8 are avalable via Support.</li><li>Auto PRs for Supply Chain findings</li></ul> |

Reviewer: ```suggestion
| Bitbucket Data Center versions 7.17 and later | <ul><li>Query console</li><li>Diff-aware scans require Bitbucket Data Center version 8.8 or later. Contact [Support](/support) for diff-aware scans if you use Bitbucket Data Center versions prior to 8.8.</li><li>Auto PRs for Supply Chain findings</li></ul> |
```
